app/src/main/kotlin/com/wire/android/ui/WireActivityDialogs.kt

```diff
@@ -363,6 +390,142 @@ fun NewClientDialog(
     }
 }
 
+@Composable
+fun CallFeedbackDialog(
+    sheetState: WireModalSheetState<Unit>,
+    onRated: (Int, Boolean) -> Unit,
+    onSkipClicked: (Boolean) -> Unit
+) {
+    val doNotAskAgain = remember {
+        mutableStateOf(false)
+    }
+    WireModalSheetLayout(
+        sheetState = sheetState,
+        sheetContent = {
+            Column(modifier = Modifier.padding(all = dimensions().spacing24x)) {
+                Text(
+                    modifier = Modifier.align(Alignment.CenterHorizontally),
+                    text = stringResource(R.string.call_feedback_dialog_title),
+                    style = MaterialTheme.wireTypography.title01
+                )
+
+                Spacer(modifier = Modifier.height(dimensions().spacing16x))
+
+                Text(
+                    modifier = Modifier
+                        .align(Alignment.CenterHorizontally)
+                        .padding(horizontal = dimensions().spacing16x),
+                    text = stringResource(R.string.call_feedback_dialog_message),
+                    style = MaterialTheme.wireTypography.body01
+                )
+
+                Spacer(modifier = Modifier.height(dimensions().spacing32x))
+
+                Row(modifier = Modifier.fillMaxWidth()) {
```

Comment:
Here you can use `horizontalArrangement = Arrangement.EqualWeight` instead of weights in all elements, or even `SpaceBetween`, `SpaceAround` or `SpaceEvenly`:
![row_arrangement_visualization](https://github.com/user-attachments/assets/0446cb5b-ba83-459e-bea4-cd1a70d71709)